src/test/java/com/ncc/aif/ExamplesAndValidationTest.java

```diff
@@ -1394,86 +1470,73 @@ void invalid() {
-                addType(relation, SeedlingOntology.GeneralAffiliation_APORA);
-                makeClusterWithPrototype(model, getClusterUri(), relation, system);
-                final Resource relationEdge = markAsArgument(model, relation,
-                        SeedlingOntology.GeneralAffiliation_APORA_Affiliate, entity, system, 1d, getAssertionUri());
-                final Resource justification = markTextJustification(model, relationEdge,
-                        "source1", 0, 4, system, 1d);
-
-                // test event edge argument is must have a compound justification
-                final Resource eventEdge = markAsArgument(model, event, SeedlingOntology.Conflict_Attack_Target,
-                        entity, system, 1.0, getAssertionUri());
-                markJustification(eventEdge, justification);
-
-
-                final Resource justification1 = makeTextJustification(model, "source1", 0, 4, system, 1d);
-                final Resource compound1 = markCompoundJustification(model,
-                        ImmutableSet.of(entity, relation, event),
-                        ImmutableSet.of(justification1),
-                        system,
-                        1d);
-
-                markJustification(entity, compound1);
-                markJustification(relation, compound1);
-                markJustification(event, compound1);
-
-                assertAndDump(model, "NIST.invalid: CompoundJustification must be used only for " +
-                                "justifications of argument assertions",
-                        nistSeedlingValidator, false);
-
+                testValid("NIST.invalid: Cluster has IRI");
             }
+
             @Test
             void valid() {
+                testValid("NIST.valid: Cluster has IRI");
+            }
+        }
+    }
 
-                // test relation argument
-                final Resource relation = makeRelation(model, getUri("relationX"), system);
-                addType(relation, SeedlingOntology.GeneralAffiliation_APORA);
-                makeClusterWithPrototype(model, getClusterUri(), relation, system);
-                final Resource relationEdge = markAsArgument(model, relation,
-                        SeedlingOntology.GeneralAffiliation_APORA_Affiliate, entity, system, 1d);
-                final Resource justification1 = makeTextJustification(model, "source1", 0, 4, system, 1d);
-                final Resource compound = markCompoundJustification(model,
-                        ImmutableSet.of(relationEdge),
-                        ImmutableSet.of(justification1),
-                        system,
-                        1d);
-
-                markJustification(relationEdge, compound);
+    /**
+     * Set of tests to show that NIST Hypothesis restrictions pass and fail appropriately
+     */
+    @Nested
+    class NISTHypothesisExamples extends BaseExample {
```

Comment:
There's some voice in my head that insists that NISTHypothesisExamples should subclass NISTExamples which should subclass BaseExample.  But there's really no reason it should right now and NISTHypothesisExamples (at least at this point) doesn't need some of the extra stuff that NISTExamples has.  But because of the NIST restriction, "Each hypothesis graph must have at least one event or relation with at least one edge," (Section 4.3 - TA3 req #5) that suggests we're going to need them soon.  Maybe we should just do it now?

Reply:
I'm not sure how that would work with @Nested tests. I'm inclined to see how much will be shared and parcel it out as appropriate. Concerns with that approach?

Reply:
Well, because of 4.3 `TA3 req #5`  we're going to need the `entity`, `event`, `relation`, `entityCluster`, `eventCluster`, and `relationCluster` members from `NISTExamples` and the `setup()` that goes with it.  We'd need to reassign the `validator` variable.

But if there are complications with @Nested tests we can at least defer the topic until we implement new hypothesis restrictions.

Reply:
Well, as a hypothesis only requires a single event or relation edge, I don't think they'd need all those things. I guess they'd avoid either the event+cluster or relation+cluster. Luckily, it sounds like JUnit has us covered: https://www.petrikainulainen.net/programming/testing/junit-5-tutorial-writing-nested-tests/. In light of this, I am removing BaseExamples and putting the functionality into the top-level class

Reply:
Funny, I was just looking at that page.  Sounds like a good approach.

Reply:
Okay, we can defer this until we have a better handle of the scope of hypothesis tests (which are probably still at least a bit in flux).  And we can weigh that against any downside to subclassing.